embed.js

```diff
@@ -0,0 +1,67 @@
+var fs = require("fs");
+var http = require("http");
+var https = require("https");
+var url = require("url");
+var express = require('express');
+var cors = require('cors');
+var app = express();
+var crypto = require('crypto');
+const path = require('path');
+app.use(cors());
+// Parse JSON bodies (as sent by API clients)
+app.use(express.json());
+
+//assign a port number for an API to run,
+const port = 8080;
+
+let appconfig;
+try {
+  appconfig = JSON.parse(fs.readFileSync('embedConfig.json'));
+} catch (error) {
+  console.error('Error: embedConfig.json file not found.');
+  process.exit(1); // Exit the program with a non-zero exit code to indicate an error
```

Comment:
process.exit(1); //Exit the program with a non-zero exit code to indicate an error.

Reply:
updated this line